components/infobox/wikis/warcraft/infobox_team_custom.lua

```diff
@@ -0,0 +1,129 @@
+---
+-- @Liquipedia
+-- wiki=warcraft
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Achievements = require('Module:Achievements in infoboxes')
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Flags = require('Module:Flags')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Center = Widgets.Center
+local Title = Widgets.Title
+
+local CustomTeam = Class.new()
+local CustomInjector = Class.new(Injector)
+
+-- These should be converted to proper links at some point
+local PROFILE_DISPLAY =
+	'<span class="league-icon-small-image" style="margin-left:2px">[[File:${icon}|32px|link=${link}|${text}]]</span>'
+local PROFILES = {
+	w3iprofile = {
+		icon = 'War3.info icon.png',
+		link = 'https://warcraft3.info/league/clans/',
+		text = 'warcraft3.info ${name}\'s profile'
+	},
+	nwc3lprofile = {
+		icon = 'NWC3L-icon.png',
+		link = 'https://nwc3l.com/team/',
+		text = 'NWC3L ${name}\'s profile'
+	},
+	eslprofile = {
+		icon = 'ESL_2019_icon.png',
+		link = 'https://play.eslgaming.com/team/',
+		text = 'ESL ${name}\'s profile'
+	},
```

Comment:
imo kick and just use `args.esl = args.esl or args.eslprofile` in `.run`

Reply:
I think that should wait until the other two also are moved to link.